ansible/server-setup/roles/nginx/files/nginx.conf

```diff
@@ -0,0 +1,127 @@
+user www-data;
+worker_processes 16;
+worker_rlimit_nofile 32000;
+pid /run/nginx.pid;
```

Comment:
Why do we need to nginx here ?

Reply:
Because some servers use nginx such as block explorer infra and blockscout